huey_monitor/tasks.py

```diff
@@ -11,6 +15,9 @@
 logger = logging.getLogger(__name__)
 
 
+DOMAIN_NAME = socket.getfqdn()
```

Comment:
Sorry, but this is _too_ early, isn't it? It runs the code on bootup of the servie. There we should not fail just because DNS is down. We should call `socket.fqdn` in a function that is cached with `functools.cache`

Reply:
good idea